docs/toml.md

```diff
@@ -1708,10 +1712,16 @@ Træfik can be configured to use Amazon ECS as a backend configuration:
 
 # ECS Cluster Name
 #
+# Deprecated - Please use Clusters
+#
+# Cluster = "default"
+
+# ECS Clusters Name
+#
 # Optional
-# Default: "default"
+# Default: ["default"]
 #
-Cluster = "default"
+Clusters = ["default"]
```

Comment:
I am confused by this change. I can't see any notion of `Clusters` in the code. Am I missing something?